pkg/taskutil/taskutil.go

```diff
@@ -38,11 +39,23 @@ import (
 // NewTask is from https://github.com/containerd/containerd/blob/v1.4.3/cmd/ctr/commands/tasks/tasks_unix.go#L70-L108
 func NewTask(ctx context.Context, client *containerd.Client, container containerd.Container,
 	flagA, flagI, flagT, flagD bool, con console.Console, logURI string) (containerd.Task, error) {
+	closer := func() {
+		if t, err := container.Task(ctx, nil); err != nil {
```

Comment:
you need `t, err := container.Task(ctx, cio.Load)` to load task with `IO`

Reply:
Thanks a ton! Only realized how obvious it is just now 😅